configure.ac

```diff
@@ -30,16 +30,39 @@ m4_define([lt_age], [3])
 m4_define([ginac_version], [ginac_major_version.ginac_minor_version.ginac_micro_version])
 m4_define([ginac_release], [ginac_major_version.ginac_minor_version])
 
-AC_INIT([pynac], ginac_version, [<pynac-devel@googlegroups.com>])
+AC_INIT([pynac], ginac_version[.sage-2020-03-28a], [<pynac-devel@googlegroups.com>])
 AC_PREREQ(2.59)
 AC_CONFIG_SRCDIR(ginac/basic.cpp)
 AC_CONFIG_HEADERS(config.h)
 dnl This defines PACKAGE and VERSION.
 AM_INIT_AUTOMAKE([gnu 1.7 dist-bzip2])
 
 AM_PATH_PYTHON([2.7])
-dnl AM_PATH_PYTHON([3.4.1])
-AX_PYTHON_DEVEL
+
+dnl Adapted from CPython 3.8 cpython/Misc/python-config.in
+AC_MSG_CHECKING([for Python preprocessor flags])
+PYTHON_CPPFLAGS=$($PYTHON - <<EOD
+import sysconfig
+print(' '.join(list(('-I' + sysconfig.get_path('include'),
+                     '-I' + sysconfig.get_path('platinclude'),))
+               + sysconfig.get_config_var('CFLAGS').split()))
+EOD)
+AC_MSG_RESULT([$PYTHON_CPPFLAGS])
+AC_SUBST([PYTHON_CPPFLAGS])
+
+AC_MSG_CHECKING([for Python library flags])
+PYTHON_LDFLAGS=$(CC=dummy-cc $PYTHON - <<EOD
+from distutils.ccompiler import new_compiler
+from distutils.sysconfig import customize_compiler
+compiler = new_compiler()
+customize_compiler(compiler)
+import sys
+import sysconfig
+assert compiler.linker_so[[0]] == 'dummy-cc'
```

Comment:
```suggestion
assert compiler.linker_so[0] == 'dummy-cc'
```

Reply:
double brackets are there because of m4.

Reply:
Ah okay. This assertion is not correct on systems with https://trac.sagemath.org/ticket/29408

Reply:
In the other script, I avoided brackets altogether, writing `list((1,2,))` instead of `[1, 2]`.
I could also do `compiler.linker_so.__getitem__(0)` here

Reply:
Sorry for the confusion. Brackets is not the issue. `linker_so` doesn't start with `dummy-cc` on systems mentioned in https://trac.sagemath.org/ticket/29408

Reply:
> https://trac.sagemath.org/ticket/29408
The change on that ticket needs more work in any case.

Reply:
The changed of the branch currently on 29408 sets LDSHARED. In that case I don't know how to reliably extract flags from this command that could be passed to libtool.

Reply:
I suppose one could just change the "assert" to a warning on stderr and hope for the best.